tests/main.test.ts

```diff
@@ -168,6 +194,16 @@ describe('#run', function () {
   });
 });
 
+async function expectError(fn: () => Promise<void>, want: string) {
```

Comment:
future: this might be helpful in `actions-utils`

Reply:
I thought about it. I actually _did_ it, but it depends on `expect` which comes from chai and then actions-utils would take a runtime dependency on a specific version of chai and... copy-paste felt easier.